src/lib/style/components/_project-view-card.scss

```diff
@@ -11,12 +11,12 @@
 
   img {
     display: block;
-    max-width: 100%;
-    max-height: 30%;
-
-    // Center the image
     margin-left: auto;
     margin-right: auto;
+    object-fit: contain;
```

Comment:
This won't work in IE: https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit